ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -837,22 +838,11 @@ export default class ConfirmTransactionBase extends Component {
   renderTitleComponent() {
```

Comment:
Nice cleanup! 

could we remove the use of `renderTitleComponent` and the `titleComponent` props? It looks like the new ConfirmTitle component you extracted supports this unless I'm missing something

Reply:
I still left the function to avoid initialising the component inline.